packages/core/src/browser/keybinding.ts

```diff
@@ -242,10 +242,8 @@ export class KeybindingRegistry {
     protected doRegisterKeybinding(binding: Keybinding, scope: KeybindingScope = KeybindingScope.DEFAULT) {
         try {
             this.resolveKeybinding(binding);
-            if (this.containsKeybinding(this.keymaps[scope], binding)) {
-                throw new Error(`"${binding.keybinding}" is in collision with something else [scope:${scope}]`);
-            }
             this.keymaps[scope].push(binding);
+            this.keybindingsChanged.fire(undefined);
```

Comment:
a bit on sure about it, on each fire menus and keybinding widget gets rerendered, it would be better to fire once for the whole keymap and fire once after all keybindings contributions are loaded (don't fire before at all)

Reply:
@marechal-p @kittaakos could you help to check whether it would affect electron perf startup

Reply:
I did some testing:
- for initial load everything is good, since no menus is rendered before all initial keybindins are loaded
- for consequent keybindings though for each binding registration top-level menus are rerendered